internal/terminal/table.go

```diff
@@ -65,8 +65,8 @@ func (t table) Message() (string, error) {
 		return "", err
 	}
 	return fmt.Sprintf(`%s
-%s
-%s
+  %s
```

Comment:
seeing as though we inject the spacer row-by-row for the `dataString`, let's do it the same way for `headerString` and `dividerString`